screenpipe-app-tauri/src-tauri/src/commands.rs

```diff
@@ -4,8 +4,69 @@
 // }
 
 use serde_json::Value;
-use tauri::Manager;
+use tauri:: Manager;
 use tracing::info;
+use tracing::{debug, error};
```

Comment:
make sure there are no duplicate keys or shortcuts added, that might cause issues (use set, or things like this)